guillotina/events.py

```diff
@@ -149,3 +150,11 @@ class NewUserAdded(object):
 
     def __init__(self, user):
         self.user = user
+
+
+@implementer(IFieldPreModified)
+class FieldPreModifiedEvent(object):
```

Comment:
Can we change the name to "BeforeFieldModifiedEvent" so it matches the scheme or our other events?